helm/coco-kafka-bridge/app-configs/kafka-bridge_delivery.yaml

```diff
@@ -0,0 +1,32 @@
+# Default values that will be applied to team clusters.
+# In team clusters, in delivery we have bridges only to the pub cluster of the team env
+
+bridges:
+# Cms bridge to pub cluster
+- name: cms-kafka-bridge-pub
+  sourceKafkaProxyUrl: "" # use the pub cluster url
```

Comment:
I was thinking to make this var configured in the config map, but if this works smoothly I'm ok with that

Reply:
We needed to pass it as Helm value, because with configMaps we can not use a value from a config map as the base value for other values